components/application/navigation.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Icon } from '@oacore/design'
+
+import styles from './navigation.css'
+
+import { Drawer } from 'design'
+
+const Navigation = ({ children, ...passProps }) => (
+  <Drawer.List {...passProps}>{children}</Drawer.List>
+)
+
+const NavigationItem = ({ children, href }) => (
+  <Drawer.Item href={href}>{children}</Drawer.Item>
+)
+
+Navigation.Item = NavigationItem
+
+const titleMap = {
+  overview: 'Overview',
+  data: 'Data',
+  statistics: 'Statistics',
+  plugins: 'Plugins',
+  settings: 'Settings',
+}
+
+const captions = Object.fromEntries(
+  ['overview', 'data', 'statistics', 'plugins', 'settings'].map(activity => [
```

Comment:
This is duplicated in many places. I know still work in progress and you will probably fix it in next commits. The reason why I am pointing this out is that maybe we could consider having some config directory where all configs (tables, routes, plugins...) would be stored. What do you think?

Reply:
Config directory is good idea. However, I am trying to avoid it as much as possible because once use have it, all your components become dependent on configs and things become broken.

It's duplicated for a reason – SRP:
- menu captions can be changed by a content manager or a designer
- urls are controlled by a developer

Here this is a shorthand mess to make it work. Captions should go somewhere to `@oacore/texts`. I am considering creating something in the dashboard for the time being and then move to separate package/repository when needed.

Reply:
Fair enough... Btw. what does SRP mean? 🤔

Reply:
It's _Single Responsibility Principle_, the first one from SOLID. It means that only one actor should control a piece of code. If there exists a feature or flow, the only one person must cause a change. Even if workflow is the same for 2 or more actors for the current moment, it should be split up because in the future these actors may have different requirements.